src/lib/data/alignment.js

```diff
@@ -131,13 +219,23 @@ export default class Alignment {
     return false
   }
 
+  /**
+   * Finds Alignment Group by token, if token is grouped in some alignment group
+   * @param {Token} token
+   * @returns {AlignmentGroup | false}

```

Comment:
Alter the return type based upon whether or not it is found seems a little odd. Isn't it more usual to return null?

Reply:
Ok - fixed